src/components/LinkAsButton/index.tsx

```diff
@@ -21,7 +21,7 @@ const LinkAsButton = (props: Props) => {
 
 /** default props for LinkAsButton component */
 const defaultProps: Props = {
-  classNameProp: 'focus-investigation btn btn-primary float-right mt-5',
+  classNameProp: 'focus-investigation btn btn-primary float-right mt-5 mr-3',
```

Comment:
What effect does this have to everyone who uses this component?  Is it viable to only add the `mr-3` to the particular button that needs the right margin?